main.go

```diff
@@ -186,6 +188,13 @@ func initialise(env EnvValues) (redisClient *redis.Client, client *gitlab.Client
 	issuesCreatedCounter = prometheus.NewCounter(issuesCreatedCounterOpts)
 	prometheus.MustRegister(issuesCreatedCounter)
 
+	issueCreationErrorCountOpts := prometheus.CounterOpts{
+		Name: "issue_creation_error_count",
```

Comment:
Can we do this for `issues_created` too? We should have consistant naming in metrics - I think what you've done is correct here, but i believe the most common suffix is `_total`

So this would give us:

- `issue_creation_errors_total`
- `issue_creation_total`

via https://prometheus.io/docs/practices/naming/#metric-names

Also, we can add the new metric to README